#: component.py:94
msgid "NextGIS geoinformation system"
msgstr "Геоинформационная система NextGIS"

#: component.py:275
msgid "Linux kernel"
msgstr "Ядро Linux"

#: component.py:278
msgid "OS distribution"
msgstr "Дистрибутив ОС"

#: component.py:290
msgid "CPU"
msgstr "Процессор"

#: component.py:293
msgid "RAM"
msgstr "ОЗУ"

#: component.py:322
msgid "Last backup"
msgstr "Последняя резервная копия"

#: component.py:326
msgid "Last maintenance"
msgstr "Последнее обслуживание"

#: exception.py:116
msgid "Validation error"
msgstr "Ошибка валидации"

#: exception.py:121
msgid "Forbidden"
msgstr "Запрещено"

#: exception.py:126
msgid "Insufficient permissions"
msgstr "Недостаточно прав"

#: exception.py:131
msgid "Operational error"
msgstr "Операционная ошибка"

#: exception.py:136
msgid "External service error"
msgstr "Ошибка внешнего сервиса"

#: exception.py:145
msgid "Not configured"
msgstr "Не сконфигурировано"

#: fontconfig.py:101
msgid "Font not found: {}!"
msgstr "Шрифт не найден: {}!"

#: storage.py:289
msgid "Storage full"
msgstr "Хранилище переполнено"

#: storage.py:290
msgid ""
"This operation can't be performed because the storage limit (%s) has been "
"already exceeded by %s."
msgstr ""
"Операция не может быть выполнена, так как лимит использования хранилища (%s) "
"уже превышен на %s."

#: storage.py:300
msgid "Not enough storage"
msgstr "Недостаточно места в хранилище"

#: storage.py:301
msgid ""
"This operation requires %s of storage and can't be performed because only %s "
"available."
msgstr ""
"Операция не может быть выполнена, так как для ее выполнения требуется %s в "
"хранилище, однако доступно только %s."

#: storage.py:311
msgid "Estimation is already running"
msgstr "Процесс оценки хранилища уже запущен"

#: test/test_exception.py:76
msgid "The answer is %d"
msgstr "Ответ %d"
